geomstats/backend/pytorch_random.py

```diff
@@ -0,0 +1,15 @@
+"""Torch based random backend."""
+
+import torch
+
+
+def rand(*args, **kwargs):
+    return torch.rand(*args, **kwargs)
+
+
+def randint(*args, **kwargs):
+    return torch.randint(*args, **kwargs)
+
+
+def seed(*args, **kwargs):
+    return torch.manual_seed(*args, **kwargs)
```

Comment:
on geomstats backend: we consistently try to be as close to numpy API everywhere, right?

Reply:
yes good tach